src/react/features/lenses/timeline/timeline.feature

```diff
@@ -2,24 +2,27 @@ Feature: Timeline lens
 
   Background:
     Given I am at home
+    When I take 'Kayaking at Potomac' in focus
     And I select timeline lens
 
-  @pending
-  Scenario: Diory markers on timeline
-    Given room 1 has 2 links with location
-    Then I see 2 markers on Timeline
+  # NOTE:
+  # 1. Timeline doesn't show marker for focus diory
+  # - although it has a date and there's no diories in view
+  # 2. Timeline shows a marker for each diory on view
+  # - although not all of them have dates
+  # 3. Currently timeline crashes if none of the diorys in view has dates
 
-  @pending
-  Scenario: Diory popup on timeline
-    When I select diory 1
-    Then I see 1 popup on timeline
+  Scenario: Shows marker for each diory on view (although not all of them have dates)
```

Comment:
I would we nice to a "Given" for diorys without date e.g.
Given focus diory 1 have 5 linked diorys without date (check from JSON?)
And focus diory 1 have 8 linked diorys with date
Then I see 0 focus and 13 linked markers on timeline

But should it show diorys without date on timeline...?